roles/image_builder/defaults/main.yml

```diff
@@ -1,21 +1,8 @@
 ---
 # defaults file for image_builder
-microshift_image_custom_repos:
-  - name: EPEL8
-    base_url: "https://dl.fedoraproject.org/pub/epel/{{ hostvars[inventory_hostname].ansible_distribution_major_version }}/Everything/x86_64/"
-    type: yum-baseurl
-    check_ssl: true
-    check_gpg: false
-  - name: microshift
-    base_url: https://download.copr.fedorainfracloud.org/results/@redhat-et/microshift-testing/epel-8-x86_64/
-    type: yum-baseurl
-    check_ssl: true
-    check_gpg: false
-  - name: microshift-deps
-    base_url: https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rpms/4.12-el8-beta/
-    type: yum-baseurl
-    check_ssl: true
-    check_gpg: false
+microshift_rhsm_repos:
```

Comment:
I think it might be a good idea to not set either of these by default and instead document in the README that if the user would like a community build they need to set `microshift_image_custom_repos` to the above values or if they want the supported version of microshift on RHEL then they need to be building against a RHEL8 system, have the correct entitlements for on their osbuild composer, and set the `microshift_rhsm_repos` like in this PR. I'm not sure if this is a good idea or not... thoughts?